### ISSUE TITLE
cannot recognize part binary

### ISSUE THREAD

**Issue by choopk** [ShaneMaglangit/agp-ts#14]
Hi do you have any idea how to fix the following error shown in the screen shot?
It happens sometimes when i fetch a new list
![bug](https://user-images.githubusercontent.com/26024002/140634108-a00c54c6-a90f-4a69-85e8-5e4c7ada5151.png)

**Comment by ShaneMaglangit** [comment]
The screenshot that you provided does not show much detail about your actual issue. That error is thrown specifically when an error occurs while parsing the binary for a body part. Its very likely that you're feeding the parser with an invalid hex representation. Thus, the parser cannot recognize it.

**Comment by choopk** [comment]
I got all the axie genes from the graphql endpoint. So could it be some of them are invalid? I am looping through all the axies to get their genes.

I followed your example in the readme to create axieGene Object.
![image](https://user-images.githubusercontent.com/26024002/140645617-2533c6a3-fdfa-4311-80d3-c3bd325b46dd.png)



![image](https://user-images.githubusercontent.com/26024002/140645933-8a79e4e8-9b6d-4fd2-8e6c-11fe3d56f875.png)

Ah i think I found out the issue. Eggs have the following hex 0x0 which caused it to fail
![image](https://user-images.githubusercontent.com/26024002/140645973-ee6710bb-b6bb-46bd-995b-baeaba5a632d.png)
